synapse/tests/test_lib_version.py

```diff
@@ -18,3 +18,148 @@ def test_version_basics(self):
         self.isinstance(s_version.verstring, str)
         tver = tuple([int(p) for p in s_version.verstring.split('.')])
         self.eq(tver, s_version.version)
+
+    def test_version_pack(self):
+        ver = s_version.packVersion(0)
+        self.eq(ver, 0)
+
+        ver = s_version.packVersion(1)
+        self.eq(ver, 1 << 64)
+        self.eq(ver, 0x000000010000000000000000)
```

Comment:
hehe, just confirming that 1 << 64 == 0x000000010000000000000000 ?  Seems like maybe the 1 << 64 one is redundant :D

Reply:
remnant of some of the first testing that was done with the delta. easy to clean up

Reply:
fixed